src/select-project.html

```diff
@@ -0,0 +1,72 @@
+
+<link rel="import" href="shared-styles.html">
+<link rel="import" href="../bower_components/vaadin-combo-box/vaadin-combo-box.html">
+<link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
+
+
+<dom-module id="select-project">
+  <template>
+    <style>
+    </style>
+
+    <!-- Fetch an array of users to be shown in the grid -->
+    <iron-ajax id="projects" url="/dummy/projects.json" handle-as="json" last-response="{{projects}}"></iron-ajax>
+
+    <vaadin-combo-box filtered-items="[[items]]" label="Project" item-label-path="title" on-filter-changed="_projectsFilter" selected-item="{{selectedProject}}">
```

Comment:
Any particular reason for using `filtered-items` instead of `items` and using a custom filtering function ?
```html
<vaadin-combo-box items="[[projects]]" label="Project" item-label-path="title" selected-item="{{selectedProject}}">
```